tools/accuracy_checker/accuracy_checker/config/config_reader.py

```diff
@@ -419,6 +419,9 @@ def merge_modules(config, arguments, update_launcher_entry):
             if value:
                 update_launcher_entry['_{}'.format(key)] = value
 
+        if arguments_dict.get('device_config'):
+            update_launcher_entry['_device_config'] = read_yaml(arguments_dict['device_config'])
```

Comment:
on my view, command line provided file should have priority under internal configuration or should be merged

Reply:
Done